invenio_cli/helpers/commands.py

```diff
@@ -243,13 +265,122 @@ def signal_handler(sig, frame):
             'docker/nginx/test.crt', '--key', 'docker/nginx/test.key'
         ], env=run_env)
 
-        click.secho('Development server + worker are running!', fg='green')
+        click.secho(
+            'Instance running!\nVisit https://localhost:5000', fg='green')
         server.wait()
 
 
 class ContainerizedCommands(object):
     """Containerized environment CLI commands."""
 
-    def __init__(self):
+    def __init__(self, cli_config, docker_helper):
         """Constructor."""
-        pass
+        self.cli_config = cli_config
+        self.docker_helper = docker_helper
+
+    def update_statics_and_assets(self, install):
+        """Update application containers' statics and assets."""
+        project_shortname = self.cli_config.get_project_shortname()
+
+        click.secho('Collecting statics and assets...', fg='green')
+        # Collect into statics/ folder
+        self.docker_helper.execute_cli_command(
+            project_shortname, 'invenio collect')
+
+        # Collect into assets/ folder
+        self.docker_helper.execute_cli_command(
+            project_shortname, 'invenio webpack create')
+
+        if install:
+            click.secho('Installing js dependencies...', fg='green')
+            # Installs in assets/node_modules/
+            self.docker_helper.execute_cli_command(
+                project_shortname, 'invenio webpack install --unsafe')
+
+        # Use the copy approach rather than the symlink one in container
+        click.secho('Copying project statics and assets...', fg='green')
+        src_dir = self.cli_config.get_project_dir() / 'static'
+        # src_dir MUST be '<dir>/.' for docker cp to work
+        src_dir = os.path.join(src_dir, '.')
+        dst_dir = '/opt/invenio/var/instance/static/'
+        self.docker_helper.copy2(src_dir, dst_dir)
+
+        src_dir = self.cli_config.get_project_dir() / 'assets'
+        # src_dir MUST be '<dir>/.' for docker cp to work
+        src_dir = os.path.join(src_dir, '.')
+        dst_dir = '/opt/invenio/var/instance/assets/'
+        self.docker_helper.copy2(src_dir, dst_dir)
+
+        click.secho('Building assets...', fg='green')
+        self.docker_helper.execute_cli_command(
+            project_shortname, 'invenio webpack build')
+
+    def _lock_python_dependencies(self):
+        """Creates Pipfile.lock if not existing."""
+        locked = 'Pipfile.lock' in os.listdir('.')
+        if not locked:
+            subprocess.run(['pipenv', 'lock'], check=True)
```

Comment:
Moderate: Here is where I would add the `--pre` :) (with `if` clause :), not enforcing it always )

Reply:
Done :+1: